src/main/java/cart/admin/view/AdminViewController.java

```diff
@@ -0,0 +1,21 @@
+package cart.admin.view;
+
+import cart.catalog.service.CatalogService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class AdminViewController {
+    private final CatalogService catalogService;
+
+    public AdminViewController(final CatalogService catalogService) {
+        this.catalogService = catalogService;
+    }
+
+    @GetMapping("")
```

Comment:
```suggestion
    @GetMapping("/admin")
```
admin 페이지 호출시 404가 뜨는데 체크해봐주세요🙏

Reply:
아앗 마지막에 합쳤다가 아닌것 같아서 분리를 했는데, 그 과정에서 url을 깜빡한것 같네요 감사합니다!